sklearn/inspection/permutation_importance.py

```diff
@@ -0,0 +1,84 @@
+"""Permutation importance for estimators"""
+import numpy as np
+
+from ..utils import check_random_state
+from ..metrics import check_scoring
+
+
+def permutation_importance(estimator, X, y, scoring=None, n_rounds=1,
+                           random_state=None):
+    """Permutation importance for feature evaluation. [BRE]_
+
+    The permutation importance of a feature is calculated as follows. First,
+    the estimator is trained on a training set. Then a baseline metric, defined
+    by ``scoring``, is evaluated on a validation set. Next, a feature column
+    from the validation set is permuted and the metric is evaluated again.
+    The permutation importance is defined to be the difference between the
+    baseline metric and metric from permutating the feature column.
+
+    Read more in the :ref:`User Guide <permutation_importance>`.
+
+    Parameters
+    ----------
+    estimator : object
+        A estimator that has already been `fit` and is compatible with
+        ``scorer``.
+
+    X : array-like or DataFrame, shape = (n_samples, n_features)
+        Training data.
+
+    y : array-like, shape = (n_samples, ...)
+        Targets for supervised learning.
+
+    scoring : string, callable or None, optional (default=None)
+        A string (see model evaluation documentation) or
+        a scorer callable object / function with signature
+        ``scorer(estimator, X, y)``.
+
+    n_rounds : int, optional (default=1)
+        Number of times to permute a feature.
```

Comment:
Nit: I think pep8 requires parameter docstring to start with "the":

The number of times...

Same for other parameters

Reply:
pep8 defined a standard for parameter docstrings? (we don't really use "the" in the beginning in other docstrings)